components/global_onboarding_list/onboarding_checklist.tsx

```diff
@@ -0,0 +1,269 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {useRef, useState, useCallback} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import styled, {css} from 'styled-components';
+
+import Icon from '@mattermost/compass-components/foundations/icon/Icon';
+
+import {FormattedMessage} from 'react-intl';
+
+import {savePreferences} from 'mattermost-redux/actions/preferences';
+import {getCurrentUserId} from 'mattermost-redux/selectors/entities/users';
+
+import {trackEvent} from 'actions/telemetry_actions';
+import checklistImg from 'images/onboarding-checklist.svg';
+import {Preferences} from 'utils/constants';
+
+import {TaskListPopover} from './onboarding_checklist_popover';
+import {Task} from './onboarding_checklist_task';
+import Completed from './onboarding_checklist_completed';
+import {CompletedAnimation} from './onboarding_checklist_animations';
+
+const TaskItems = styled.div`
+    border-radius: 4px;
+    border: solid 1px rgba(var(--center-channel-color-rgb), 0.16);
+    background-color: var(--center-channel-bg);
+    max-width: 352px;
+    padding: 1rem 0;
+    transform: scale(0);
+    opacity: 0;
+    box-shadow: 0px 20px 32px rgba(0, 0, 0, 0.12);
+    transition: opacity 150ms ease-in-out 0ms, transform 150ms ease-in-out 0ms;
+    transform-origin: left bottom;
+    height: 518px;
```

Comment:
Can we keep the height dynamic as there will be cases where it’s possible we will have less items in the list our more, Like I added Playbooks item in the list scrollbar appeared

Reply:
ill convert to max-height in order to keep the wrapper on the next page consistent.